src/main.js

```diff
@@ -92,6 +95,8 @@ function compileTo(src, dest) {
 
 if (argv.interactive) {
     repl.start();
+} else if (argv._.length === 1) {
+    compileTo(argv._[0], undefined);
```

Comment:
It's kinda weird to use this function to either compile or eval. How about splitting this function into one that returns the compiled code and call it `compile`, then have like `compileAndEval` and `compileToDest` use it in this if/else area?